modules/virt-customizing-storage-profile-default-cloning-strategy.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-customizing-storage-profile-default-cloning-strategy_{context}"]
+= Setting a default cloning strategy by using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creating a _cloning strategy_. Setting cloning strategies can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
```

Comment:
Is there a reason cloning strategy is italicized here?

Reply:
I don't know, as I didn't update that content. I've removed the italics.